api-service/cloudify_api/config.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseSettings
```

Comment:
What does pydantic.BaseSettings buy us in exchange for having this extra dependency?

Reply:
pydantic is required by fastapi anyway; it's the model/schema library that it uses for parsing request bodies etc

Reply:
Yeah, so at the moment we're not leveraging the parsing/validation features of `BaseSettings`.  That's the plan for the future, when we get rid of `manager_rest.config` dependency.
About the cost, it's probably negligible, because pydantic is already fastapi's dependency, and we cannot get rid of it anyway.

Reply:
OK, sounds good then.